core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/EntityComponent.scala

```diff
@@ -787,6 +811,15 @@ trait EntityComponent {
       }
     }
 
+    def doesEntityTypeAlreadyExist(workspaceContext: Workspace, entityType: String): ReadAction[Seq[Boolean]] = {
+      CheckForExistingEntityTypeQuery.doesEntityTypeAlreadyExist(workspaceContext, entityType)
+    }
```

Comment:
optionally, you could use `uniqueResult` here:
```scala
def doesEntityTypeAlreadyExist(workspaceContext: Workspace, entityType: String): ReadAction[Option[Boolean]] = {
  uniqueResult(CheckForExistingEntityTypeQuery.doesEntityTypeAlreadyExist(workspaceContext, entityType))
}
```
I find it easier to think in terms of Options for a result like this than a Seq, where callers will always only be looking at Seq.head. But they're functionally equivalent; this is more of a code-style comment.

Thinking out loud: we could also add an `exactlyOneResult` helper parallel to `uniqueResult` - whereas `uniqueResult` expects 0 or 1 rows, `exactlyOneResult` would expect exactly 1.

Reply:
I've changed to `uniqueResult`.